docs/user/ovirt/customization.md

```diff
@@ -0,0 +1,55 @@
+# oVirt IPI customization
+
+Before you begin see the [generic cluster customization doc](docs/user/customization.md) 
+
+Installation on oVirt allows various customization of the created resources. 
+Those changes are made prior to the cluster install as they will affect the  
+creation of the resources, and we want to make sure all of the resources have 
+the same set of definitions.
+ 
+The installer-created resources are:
+- RHCOS qcow image (uploaded on cluster creation) 
+- Template from the pinned RHCOS qcow disk
+- master Vms, and disks, from the template.
+
+A dedicated VLAN network per cluster may come in the future. 
+
+Customization can be made in prior to the installation or after the cluster is created.
+ 
+## Customize MEM/CPU
+
+Env variables for the install time:
+
+| Env variable  | Description |
+| ---  | --- |
+| TF_VAR_ovirt_template_disk_size_gib | The size of the template disk for worker/nodes in GiB | 
+| TF_VAR_ovirt_master_mem | Master VM memory in MiB  | 
+| TF_VAR_ovirt_master_cpu | Master number of CPUs | 
+| TF_VAR_ovirt_template_mem | Template VM memory in MiB | 
+| TF_VAR_ovirt_template_cpu | Template number of CPUs |
```

Comment:
These options are only available via TF_VARs? We shouldn't be suggesting that to users only developers, can these be set on the machine pools?